content/geoip/updating-databases.mdx

```diff
@@ -124,8 +126,9 @@ In order to download the databases from a script or program, please use the
 permalinks found on the
 [GeoIP download page](https://www.maxmind.com/en/accounts/current/geoip/downloads).
 Please note that you will be redirected from these permalinks because MaxMind
-uses R2 presigned URLs for database downloads. You should make sure that your
-servers can make HTTPS connections to the following hostname:
+uses R2 presigned URLs for database downloads. This means that we will begin
```

Comment:
The "This means" sentence seems unnecessary to me. If we want to keep it, should we rephrase it to state that we do redirect? Once we've fully rolled out redirects, we wouldn't want it to say "we will begin" at least.